### PR TITLE
PY-16945: Support .coveragerc file

### PR DESCRIPTION
**Pull request by GuillaumeCisco**
Fix https://youtrack.jetbrains.com/issue/PY-16945

### CODE REVIEW

**Comment by traff** [comment]
Thanks for the fix!

**Comment by GuillaumeCisco** [comment]
Thanks @traff.
I don't understand, what should I add as an improvement?

File: python/helpers/coverage_runner/run_coverage.py [hunk 1]
```diff
@@ -46,14 +46,19 @@
 cwd = os.getcwd()
 
 run_xml = os.getenv('PYCHARM_RUN_COVERAGE_XML')
+argv = ["xml", "-o", coverage_file + ".xml", "--ignore-errors"]
+rcfile = cwd + "/.coveragerc"
+if os.path.exists(rcfile):
+    print("Loading rcfile\n")
```
[the remark this answers is not in the document]
Comment (GuillaumeCisco) [reply python/helpers/coverage_runner/run_coverage.py]: just did :)

**Comment by traff** [comment]
Sorry, for some reason my comment is lost. Could you please write the path of the rcfile that is loaded to the message that is printed.

**Comment by GuillaumeCisco** [comment]
@traff Your comment is not lost, I've just clicked on "resolve the conversation" as I added the path of the rcfile to the printed message. I amended my previous commit, for avoiding making two commits. You can check the actual code ;)

**Comment by traff** [comment]
Thanks! Merged

**Pull request closed without merging**